ocs_ci/helpers/dr_helpers.py

```diff
@@ -286,22 +291,49 @@ def relocate(
     config.switch_ctx(restore_index)
 
 
-def check_mirroring_status_ok(replaying_images=None):
+def check_mirroring_status_ok(
+    replaying_images=None, cephblockpoolradosns=None, storageclient_uid=None
+):
     """
     Check if mirroring status has health OK and expected number of replaying images
 
     Args:
         replaying_images (int): Expected number of images in replaying state
+        cephblockpoolradosns (string): The name of the cephblockpoolradosnamespace
+        storageclient_id(string): The uid of the storageclient in the client cluster where the application is running.
+            Applicable for provider - client configuration.
 
     Returns:
         bool: True if status contains expected health and states values, False otherwise
 
+    Raises:
+        NotFoundError: If the configuration is provider mode and the name of the cephblockpoolradosnamespace
+            is not obtained
     """
-    cbp_obj = ocp.OCP(
-        kind=constants.CEPHBLOCKPOOL,
-        resource_name=constants.DEFAULT_CEPHBLOCKPOOL,
-        namespace=config.ENV_DATA["cluster_namespace"],
-    )
+    if is_hci_cluster():
+        logger.info("Get the cephblockpoolradosnamespace associated with storageclient")
+        cephbpradosns = (
+            config.ENV_DATA.get("radosnamespace_name", False) or cephblockpoolradosns
+        )
+        if not cephbpradosns:
+            cephbpradosns = find_cephblockpoolradosnamespace(
+                storageclient_uid=storageclient_uid
+            )
+
+        if not cephbpradosns:
+            raise NotFoundError("Couldn't identify the cephblockpoolradosnamespace")
+
```

Comment:
I think there in an indentation error, the 2nd "if not cephbpradosns" will be inside the 1st if condition

Reply:
Removed the first if condition